docs/source/cuml-accel/limitations.rst

```diff
@@ -443,6 +443,40 @@ Additionally, the following fitted attributes are currently not computed:
 
 - ``n_iter_``
 
+LinearSVC
+^^^^^^^^^
+
+``LinearSVC`` will fall back to CPU in the following cases:
+
+- If ``X`` is sparse.
+
+The following fitted attributes are currently not computed:
+
+- ``n_iter_``
+
+Additional notes:
+
+- Sample weight functionality may not produce equivalent results to replicating data according to weights.
+- Models may not be picklable; pickling or unpickling may fail.
+- Error messages and formats may differ from scikit-learn, including C++ stack traces.
+- Multi-class models may have coefficient shape differences that cause pickling failures.
+
+LinearSVR
+^^^^^^^^^
+
+``LinearSVR`` will fall back to CPU in the following cases:
+
+- If ``X`` is sparse.
+
+The following fitted attributes are currently not computed:
+
+- ``n_iter_``
+
+Additional notes:
+
+- The ``coef_`` attribute returns a 2D array with shape ``(1, n_features)`` instead of the expected 1D array with shape ``(n_features,)``.
```

Comment:
Can we fix this? Should be fairly straightforward in `_attrs_to_cpu`/`_attrs_from_cpu` to massage the shape (if it's hard to change the internal cuml representation).

Reply:
At least partially addressed in 5fce26e5f21a5d37cf7171262de5cecbdee2fb6f .